display.js

```diff
@@ -0,0 +1,17 @@
+export function displayList() {
+  document.querySelector('#book-list').style.display = 'block';
+  document.querySelector('#book-form').style.display = 'none';
+  document.querySelector('#contact').style.display = 'none';
+}
+
+export function displayForm() {
+  document.querySelector('#book-list').style.display = 'none';
+  document.querySelector('#book-form').style.display = 'block';
+  document.querySelector('#contact').style.display = 'none';
+}
+
+export function displayContact() {
+  document.querySelector('#book-list').style.display = 'none';
+  document.querySelector('#book-form').style.display = 'none';
+  document.querySelector('#contact').style.display = 'block';
+}
```

Comment:
-  [ ] Good job so far, :+1: :+1: I think there are some improvements you need to make on these functions according to the project requirements.
> Refactor your functions to arrow functions.

- - According to the project requirement, you are meant to refactor all your functions to use the ES6 arrow function, I'll suggest you follow the requirement and refactor these functions. :+1: :smile: